e2e/tests/performance/memleak-imagery.perf.spec.js

```diff
@@ -57,15 +57,14 @@ test.describe.only('Memory Performance tests', () => {
     await page.waitForTimeout(3 * 1000);
     await forceGC(page);
     await captureHeapSnapshot(page, snapshotPath + 's3.heapsnapshot');
+
     // Get and compare JSHeapUsedSize at different points in the test
     let heapSize = await getHeapSize(page);
     console.log(`Final JSHeapUsedSize: ${heapSize}`);
 
-    console.log('Heap snapshot captured');
     const reader = BrowserInteractionResultReader.from(snapshotPath);
```

Comment:
It's almost there. The `path` passed to `BrowserInteractionResultReader.from(path);` should be `e2e/test-data/heapsnapshots` rather than `e2e/test-data/heapsnapshots/data/cur/`